dashboard/src/utility/dashboardClasses.js

```diff
@@ -0,0 +1,9 @@
+import classNames from 'classnames'
+import snakeCase from 'lodash/snakecase'
+
+const dashboardClasses = (owner, ...args) => {
```

Comment:
I think this all pretty much looks good, but some aesthetic/bikesheddy things to consider briefly, before we land:

1. `dashboardClasses` is a bit ambiguous as a name. Strawman: `componentClassNames`, which implies that you should use it for the `classNames` prop for components. In any case, a descriptive comment for the function would also help legibility of that pattern.
2. There are a few things we could do to the component name: snake_case them, leave them alone, prefix them, or some combination of those. snake_case feels a bit neither-here-nor-there to me, because it is a Rails-idiomatic transformation of the original name; also, CSS class names are traditionally kebab-case. If we wanted to be super verbose and explicit, we could do something like `component-OriginalComponentName`. Here, the original component class is preserved without transformation, but there is a prefix, to prevent ambiguous collisions. Anyway, YMMV.
3. I don't think `owner` should ever be allowed to be null. In fact, we could even consider adding a pseudo-static check to make sure that `owner` derives from `React.Component`.